docs/validation-server/DESIGN.md

```diff
@@ -0,0 +1,27 @@
+The Plutus Core validation server will behave as follows:
+
+  - Bind to port 3000
+
+  - Accept HTTP requests with a JSON body
+
+  - The server should accept `GET` requests that contain the JSON data for
+    validation in the request body
```

Comment:
This could do with a bit more detail about what the type of the data actually is. I don't think we need to replicate the datatype definition, but just something like "the validator, redeemer, and data scripts as base-64-encoded binary data, and the validation info as ???".